build/Dockerfile

```diff
@@ -1,35 +1,47 @@
-# build stage for building binaries
-FROM golang:1.13.8 as build
+# We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
+# to install it.
+FROM registry.access.redhat.com/ubi8/ubi-minimal as build
 LABEL stage=build
+RUN microdnf -y install rsync make git tar findutils diffutils
+RUN mkdir /build/
+
+# Install go 1.14
```

Comment:
With this we are using go 1.14 to build WMCO. Shouldn't we also reflect this in WMCO's `go.mod` in addition to the [release repo](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml#L6)?